### PR TITLE
Isecicd272&273: Fix import task template with empty command and 403 error message

### PR DESCRIPTION
**Pull request by Amy0511**
## Context

Related Issues(s): 

Build Number: 

## Checklist:

- [ ] Verified in an integrated environment
- [ ] Unit and integration tests passing
- [ ] Semantic HTML with accessibility support
- [ ] No linting, test console, or browser console errors (best effort)
- [ ] JSDoc comment blocks for complex code

### CODE REVIEW

**Comment by Amy0511** [comment]
@timrbula @BenjaminRuby Could you please help to review the code? It fixed the bug of import task template with empty command error and also fixed the 403 error message.

**Review by shenqinb-star — APPROVED** [review]

File: src/Features/TaskTemplates/Sidenav/AddTaskTemplate/AddTaskTemplate.tsx [hunk 1]
```diff
@@ -43,12 +43,20 @@ function AddTaskTemplate({ addTemplateInState, taskTemplates, history, location
       closeModal();
     } catch (err) {
       if (!isCancel(err)) {
+        let errorTitle = err.toString();
```
Comment (timrbula) [inline src/Features/TaskTemplates/Sidenav/AddTaskTemplate/AddTaskTemplate.tsx:46]: We have a `formatErrorMessage` util, did yall try using that for formatting the error? 

`import { formatErrorMessage } from "@boomerang-io/utils";`

File: src/Features/ManageTeamTasks/Sidenav/AddTaskTemplate/AddTaskTemplate.tsx [hunk 2]
```diff
@@ -45,12 +45,20 @@ function AddTaskTemplate({ addTemplateInState, taskTemplates, history, location
       closeModal();
     } catch (err) {
       if (!isCancel(err)) {
+        let errorTitle = err.toString();
```
Comment (timrbula) [inline src/Features/ManageTeamTasks/Sidenav/AddTaskTemplate/AddTaskTemplate.tsx:48]: We have a `formatErrorMessage` util, did yall try using that for formatting the error? 

`import { formatErrorMessage } from "@boomerang-io/utils";`

**Review by timrbula — CHANGES_REQUESTED** [review]
Added the same question twice about formatting the error message. Otherwise, things look good

**Review by shenqinb-star — APPROVED** [review]

**Comment by Amy0511** [comment]
@timrbula Could you please help to review it again? I changed to use `formatErrorMessage`. Thanks!

**Review by timrbula — APPROVED** [review]

**Pull request merged**